app/models.py

```diff
@@ -1820,6 +1833,35 @@ def serialize(self):
 
         return data
 
+    def create_just_in_time_assessment_tasks(self):
+        supplier = self.supplier
+        assessed_domains = supplier.assessed_domains
+
+        if assessed_domains:
+            return
+
+        if current_app.config['JIRA_FEATURES'] and \
+                current_app.config['JUST_IN_TIME_ASSESSMENTS']:
+
+            mj = get_marketplace_jira()
+
+            domain = self.brief.domain
+
+            if domain:
+                # brief has an associated domain, assess this one
+                domains = [domain]
+            else:
+                # create assessment task for each domain
+                domains = [
```

Comment:
we will need to clarify this logic - there are also other scenarios we will need to account for